frontend/src/components/common/EditorDialog.tsx

```diff
@@ -292,16 +292,28 @@ const useStyles = makeStyles(theme => ({
 function DocsViewer(props: {docSpecs: any}) {
   const { docSpecs } = props;
   const classes = useStyles();
+  const docsErrorCommonString = 'Cannot load documentation:';
```

Comment:
I understand why you used a var here, but this is a message and will likely get translated later. As a rule one shouldn't do things like concatenation of translatable strings programatically like this patch does, so maybe just repeat the text.